jdbc/src/main/java/io/kestra/jdbc/runner/JdbcQueue.java

```diff
@@ -69,6 +73,7 @@ public abstract class JdbcQueue<T> implements QueueInterface<T> {
     public JdbcQueue(Class<T> cls, ApplicationContext applicationContext) {
         ExecutorsUtils executorsUtils = applicationContext.getBean(ExecutorsUtils.class);
         this.poolExecutor = executorsUtils.cachedThreadPool("jdbc-queue-" + cls.getSimpleName());
+        this.asyncPoolExecutor = executorsUtils.maxCachedThreadPool(MAX_ASYNC_THREADS, "jdbc-queue-async-" + cls.getSimpleName());
```

Comment:
Are we sure that poll will not rejected if there is too many thread, the configuration had me some doubt.

Reply:
No, I used a single thread executor at first and nothing was refused.
An executor always accepts new tasks but if there is no thread it will add it to a queue to be processed later.

Reply:
Heeee, no it's a multiple one as I see, using 2 * cpu count as max :thinking:

Reply:
Yes, I said at first, now I've moved to more threads as it's doing I/O.
Maybe we didn't understand clearly each other, you're afraid that:
- A log event could be rejected? As far as i know this can only occurs during executor shutdown which is normal and I think is already the case
- The executor cannot start that much thread? This is a maximum of threads and we use unbounded executor in other places which is worst.

Reply:
thanks for clarification, so ok